Resources/Prototypes/Entities/Clothing/Head/sindicap.yaml

```
@@ -0,0 +1,13 @@
+- type: entity
```

```suggestion
- type: entity
```
Почему у нас фаил расширением .yaml? У нас все файлы .yml